Assets/TabMasterScript.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Microsoft.MixedReality.Toolkit.UI;
+using UnityEngine;
+
+public class TabMasterScript : MonoBehaviour
+{
+    public tab1 tab_one;
```

Comment:
Almost any time that you use `X1` `X2`... `X#`, it should almost always be an array, list, or set named `XList`

Reply:
or `XSet`